src/qps/interface/qps.c

```diff
@@ -1160,75 +1163,46 @@ PetscErrorCode QPSMonitor(QPS qps, PetscInt it, PetscReal rnorm)
 
    Input Parameters:
 +  qps - iterative context obtained from QPSCreate()
-.  monitor - pointer to function (if this is NULL, it turns off monitoring
-.  mctx    - [optional] context for private data for the
-              monitor routine (use NULL if no context is desired)
--  monitordestroy - [optional] routine that frees monitor context
-                    (may be NULL)
+.  monitor - pointer to function
+.  mctx    - [optional] context for private data for the monitor routine (use NULL if no context is desired)
+-  monitordestroy - [optional] routine that frees monitor context (may be NULL)
 
    Calling Sequence of monitor:
-$     monitor (QPS qps, int it, PetscReal rnorm, void *mctx)
+$     monitor (QPS qps, PetscInt it, PetscReal rnorm, void *mctx)
 
 +  qps - iterative context obtained from QPSCreate()
 .  it - iteration number
-.  rnorm - (estimated) 2-norm of (preconditioned) residual
+.  rnorm - estimated 2-norm of the residual
```

Comment:
When it is estimated?

Strictly speaking, we usually do not deal with a plain residual but I do not have a better wording.

Reply:
Maybe approximate is better word?

I would say we use the word residual quite consistently as a vector that is evaluated for convergence.

```suggestion
.  rnorm - approximate 2-norm of the residual (more general, a vector whose norm determines convergence, such as a projected gradient for QPSMPGP)
```

Reply:
But the latter probably appears multiple times across PERMON.

Reply:
My point was, that when we compute rnorm it is probably never estimated or approximated.

Reply:
Well, everything is approximate on computers ;-)
```suggestion
.  rnorm - 2-norm of the residual (more general, a vector whose norm determines convergence, such as a projected gradient for QPSMPGP)```